### PR TITLE
Instance actions updates

### PR DESCRIPTION
**Pull request by MarkWoulfeNeo4j**

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
Looks like you've updated the documentation!

Check out your changes at https://neo4j-docs-aura-247.surge.sh

**Comment by MarkWoulfeNeo4j** [comment]
[Document - 'Upgrade to Professional' CTA](https://trello.com/c/MuSXVipF/203-document-upgrade-to-professional-cta)

File: modules/ROOT/pages/auradb/managing-databases/database-actions.adoc [hunk 1]
```diff
@@ -15,24 +15,11 @@ To rename an instance:
 . Enter a new name for the instance.
 . Select *Rename*.
 
-== Loading a dataset
```
Comment (nvitucci) [inline modules/ROOT/pages/auradb/managing-databases/database-actions.adoc:18]: You can no longer do this?

Comment (MarkWoulfeNeo4j) [reply modules/ROOT/pages/auradb/managing-databases/database-actions.adoc:18]: nope, datasets were deleted from console, now a workspace thing.

**Review by nvitucci — APPROVED** [review]

**Pull request merged**